app/application.py

```diff
@@ -115,4 +115,4 @@ def log_request():
         args_list.append('arg ' + str(count) + ' = ' + str(key) + ': ' + str(val))
 
     params = ''.join(args_list)
-    logger.debug('Headers: ' + str(headers) + ' Body: ' + str(req_data) + ' Arguments: ' + str(params))
+    logger.debug('Incoming request', headers=headers, req_data=req_data, arguments=params)
```

Comment:
Being that this is secure messaging, is there anything (personally identifying info etc) in the `headers`, `req_data` or `params`?